src/components/movie/movie-card/movie-card.jsx

```diff
@@ -1,14 +1,61 @@
+import "./movie-card.scss";
 import PropTypes, { shape } from "prop-types";
+import { Card, Button, Row, Col } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+export const MovieCard = ({
+  movie,
+  user,
+  method,
+  messageOK,
+  messageFailed,
+  fav,
+}) => {
+  const favHandle = () => {
+    const storedToken = localStorage.getItem("token");
+    fetch(
+      `https://movies-flix-al-f68cdd84f041.herokuapp.com/users/${user.Username}/movies/${movie._id}`,
+      {
+        method: method,
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${storedToken}`,
+        },
+      }
+    ).then((response) => {
+      if (response.ok) {
+        alert(messageOK);
+      } else {
+        alert(messageFailed);
+      }
+    });
+  };
```

Comment:
After implementing A & B, you no longer need this function. You will use the other three props that were added in B.